recipe/meta.yaml

```diff
@@ -43,6 +45,7 @@ requirements:
     - cmake  <3.22
```

Comment:
Should we rely on cmake <3.22 or use the latest version available?

Reply:
We will have to use the pinning for `cmake <3.22` for now, when we use the latest version of `cmake` the build fails.